pkg/scheduler/partition.go

```diff
@@ -324,12 +324,12 @@ func (pc *PartitionContext) AddApplication(app *objects.Application) error {
 		if isRecoveryQueue {
 			queue, err = pc.createRecoveryQueue()
 			if err != nil {
-				return fmt.Errorf("failed to create recovery queue %s for application %s", common.RecoveryQueueFull, appID)
+				return fmt.Errorf(strings.Join([]string{"failed to create recovery queue", queueName, "for application", appID, ":", err.Error()}, " "))
```

Comment:
I think it should be error.join(), as mentioned in the jira comment section.

an quick example:
```go
package main

import (
	"errors"
	"fmt"
)

func main() {
	err1 := errors.New("err1")
	err2 := errors.New("err2")
	err := errors.Join(err1, err2)
	fmt.Println(err)
	if errors.Is(err, err1) {
		fmt.Println("err is err1")
	}
	if errors.Is(err, err2) {
		fmt.Println("err is err2")
	}
}
```

Reply:
Thank you for your feedback. Let me take a moment to reconsider the meaning of the message.

Reply:
I’ve finished modifying the PR, but I’m not sure if I have correctly expressed it in the Jira comment section. Please take a look.

Reply:
I think it does! Although I never tested it. 
Great job!